groups/assemble/assemble_unsigned_psbt.js

```diff
@@ -83,9 +83,15 @@ module.exports = ({capacity, proposed, rate}, cbk) => {
 
           const inputsOffset = tx.ins.length;
 
-          [member.change, member.funding].filter(n => !!n).forEach(out => {
-            return tx.addOutput(hexAsBuffer(out), capacity);
-          });
+          if (!!member.change) {
+            tx.addOutput(hexAsBuffer(member.change), capacity);
+          }
+
+          if (!!member.funding && !!member.funding.length) {
```

Comment:
is there a case when there isn't member funding? that seems like a problem if you are just making change

Reply:
that check was already sort of there before for group open where we filter out null values in the array. so i am just checking the same thing.

https://github.com/alexbosworth/paid-services/blob/master/groups/assemble/assemble_unsigned_psbt.js#L86

Reply:
and i think it is undefined in some case for group_open. i am not sure what the case is. maybe for pair groups.